designsafe/apps/api/views.py

```diff
@@ -155,3 +156,41 @@ def post(self, request):
             },
         )
         return HttpResponse("OK", status=202)
+    
+
+
+class SystemMonitorProxyApi(BaseApiView):
+    """"
+    Proxy API for fetching system queue data from Core Portal. 
+    """
+
+    def get(self, request, hostname):
+        CORE_PORTAL_API_BASE = "https://cep.tacc.utexas.edu/api/system-monitor"
+
+        url = f"{CORE_PORTAL_API_BASE}/{hostname}"
+        print(f"Proxy API calling: {url}") 
+
+        
+        try:
+            response = requests.get(url, timeout=10)
+            response.raise_for_status()
+            return JsonResponse(response.json(), safe=False)
+        except requests.exceptions.RequestException as e:
+            print("Proxy API Error:", str(e)) 
+            return JsonResponse({"error": str(e)}, status=500)
+            
+
+class SystemOverviewProxyApi(BaseApiView):
+    """
+    Proxy API for fetching system monitor overview data from Core Portal. (Load%, Running, Waiting Jobs)
+    """
+    def get(self, request):
+        CORE_PORTAL_API_BASE = "https://cep.tacc.utexas.edu/api/system-monitor"
```

Comment:
We usually store variables like this in the common_settings.py file, so that they can be easily kept track of. You can see it in the CEP version (we call it settings.py there) -- https://github.com/TACC/Core-Portal/blob/c210828226c8c587a76f18e7ae5d8a849adf3a48/server/portal/apps/system_monitor/views.py#L28